### ISSUE TITLE
Can't add device to Homeassistant. Skip setup for encrypted device

### ISSUE THREAD

**Issue by dshulinin** [AlexxIT/SonoffLAN#993]
Hi all.
I've setup the SonoffLAN with my username and password for EWElink. In the EWElink app i have one device.
However I can't import it into the Homeassistant.
The debug log shows the following:

2022-10-01 12:43:24 [D] SysInfo: {'installation_type': 'Home Assistant Supervised', 'version': '2022.6.7', 'dev': False, 'hassio': True, 'virtualenv': False, 'python_version': '3.9.12', 'docker': True, 'arch': 'x86_64', 'timezone': 'Europe/Moscow', 'os_name': 'Linux', 'os_version': '4.19.0-14-amd64', 'user': 'root', 'supervisor': '2022.08.6', 'host_os': 'Debian GNU/Linux 10 (buster)', 'docker_version': '20.10.3', 'chassis': 'desktop', 'sonoff_version': '3.3.1 (b20e33c)'}
2022-10-01 12:43:29 [D] AUTO mode start
2022-10-01 12:43:30 [D] 10010100c4 !! skip setup for encrypted device

Have also tried local and cloud modes with no luck either.
Please help!

The device firmware on EWElink account shows 3.5.1. The SonoffLAN i'm using is the most current version - 3.3.1

**Comment by MrBasque** [comment]
Same issue here, have 2 sonoff dualR3 devices, working fine but are not added to home assistant. 

Debug log says: skip setup for encrypted device

**Comment by AlexxIT** [comment]
**skip setup for encrypted device** - means that a device is found on the local network, but it is not in your cloud account.

It could happened:
- you did not enter your username/password
- you entered username/password from another account
- the device is linked to another Home
- You're having trouble connecting to the cloud

**Comment by giosooul** [comment]
Good morning, 
I have the same issue:

2023-02-17 13:24:27 [D] 1001xxxxx9 !! skip setup for encrypted device

I confirm that the user and password are correct and that I can see the device from eWeLink app (it is a smart circuit breaker what was working with the integration).

**Comment by AlexxIT** [comment]
There is can't be any magic. Problem device not in your current account and home.

**Comment by Adam-Dalloul** [comment]
same issue on the 3.6 firmware that was recently released. any updates ?

**Comment by yousaf465** [comment]
same issue here total of 11 devices. 2 added recently but not being shown in SOnoff 
logs say

```skip setup for encrypted device```

these were smart bulb holders and, the same model of device had already worked fine .

tried to add 11 devices again, and got this error.

**Comment by AlexxIT** [comment]
You should restart Hass for download latest device list

**Comment by yousaf465** [comment]
I did tried reloading the intergration

**Comment by AlexxIT** [comment]
There is instructions in the docs, how you should debug your problem

**Comment by yousaf465** [comment]
I had to restore 2023.11.3 from 2023.12.b0. all 11 devices were added magically. 
Only issue is even though the new devices are bulb holder but don't show up as such, only mentioned as switches, even though I have chosen display as light in ewelink app 
old smart bulb holder 
![image](https://github.com/AlexxIT/SonoffLAN/assets/83491212/14d8a215-add9-4adb-8145-557d93ddf504)
new bulb holder, had to use switch as X helper in Homeassistant. 
![image](https://github.com/AlexxIT/SonoffLAN/assets/83491212/c42732d0-62b0-4500-aeff-b9cd0eefa323)
